src/nvim/eval.c

```diff
@@ -11737,25 +11737,28 @@ static void f_jobstart(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 }
 
 // "jobstop()" function
-static void f_jobstop(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+static int f_jobstop(typval_T *argvars, typval_T *rettv, FunPtr fptr)
```

Comment:
the signature is always `void` no matter the return type. Instead you would need to change `rettv->vval.v_number`. But please describe the change in words, and what it is needed for.

Reply:
Could you just briefly describe me, what do I need to change ?

Reply:
No, because it is not clear what you want to do and what it is useful for. Process exit status (or signal) is already available in on_exit callback. IIRC `jobstop()` will not wait until the process is exited, so it might not even be available yet.